cli/src/deploy/mod.rs

```diff
@@ -795,10 +761,43 @@ pub async fn terminate(cluster_id: &str) {
   sp.finish_with_message(&res);
 }
 
-pub async fn new(body: Value) {
+pub async fn new(
+  agz_b64: String,
+  anycloud_params: Option<(String, String)>,
+  env_b64: Option<String>,
+) {
```

Comment:
This change un-DRYs work we did before to make it easy to have different behavior between `anycloud new` and `alan deploy`. How will this work on the `alan deploy` side?

Reply:
alan deploy will call this method with `None` for the last two params

Reply:
A bit less flexible, but it should work.